pkg/daemon/update.go

```diff
@@ -1598,12 +1597,25 @@ func (dn *Daemon) atomicallyWriteSSHKey(authKeyPath, keys string) error {
 }
 
 // Set a given PasswdUser's Password Hash
-func (dn *Daemon) SetPasswordHash(newUsers []ign3types.PasswdUser) error {
+func (dn *Daemon) SetPasswordHash(newUsers, oldUsers []ign3types.PasswdUser) error {
 	// confirm that user exits
-	if len(newUsers) == 0 {
-		return nil
+	glog.Info("checking if absent users need to be disconfigured")
```

Comment:
with PR https://github.com/openshift/machine-config-operator/pull/3734  we are now using only klog, so will be good to switch these as well.